doc/manual/io.rst

```diff
@@ -7,39 +7,83 @@ This guide describes how to import data into MNE. It includes instructions for
 importing from files generated by common MEG/EEG recording equipment, as well
 as importing raw data from NumPy arrays.
 
-.. contents:: Contents
+.. contents:: Page contents
    :local:
    :depth: 2
 
 .. _data-formats:
 
-Cheatsheet
-##########
-
-The cheatsheet below summarizes the supported file formats.
-
-===================   ============================  =========  =================================================================
-Datatype              File format                   Extension  MNE-Python function
-===================   ============================  =========  =================================================================
-MEG                   Elekta Neuromag               .fif       :func:`mne.io.read_raw_fif`
-MEG                   4-D Neuroimaging / BTI         dir       :func:`mne.io.read_raw_bti`
-MEG                   CTF                            dir       :func:`mne.io.read_raw_ctf`
-MEG                   KIT                            sqd       :func:`mne.io.read_raw_kit` and :func:`mne.read_epochs_kit`
-EEG                   Brainvision                   .vhdr      :func:`mne.io.read_raw_brainvision`
-EEG                   Neuroscan CNT                 .cnt       :func:`mne.io.read_raw_cnt`
-EEG                   European data format          .edf       :func:`mne.io.read_raw_edf`
-EEG                   Biosemi data format           .bdf       :func:`mne.io.read_raw_bdf`
-EEG                   General data format           .gdf       :func:`mne.io.read_raw_gdf`
-EEG                   EGI simple binary             .egi       :func:`mne.io.read_raw_egi`
-EEG                   EGI MFF format                .mff       :func:`mne.io.read_raw_egi`
-EEG                   EEGLAB                        .set       :func:`mne.io.read_raw_eeglab` and :func:`mne.read_epochs_eeglab`
-EEG                   eXimia                        .nxe       :func:`mne.io.read_raw_eximia`
-Electrode locations   elc, txt, csd, sfp, htps      Misc       :func:`mne.channels.read_montage`
-Electrode locations   EEGLAB loc, locs, eloc        Misc       :func:`mne.channels.read_montage`
-MEG and EEG           FieldTrip raw/epochs/avg      .mat       :func:`mne.io.read_raw_fieldtrip`, :func:`mne.read_epochs_fieldtrip` and :func:`mne.read_evoked_fieldtrip`
-MEG and EEG           arbitrary data (from memory)  xxxx       :class:`mne.io.RawArray`, :class:`mne.EpochsArray`, :class:`mne.EvokedArray` and :func:`mne.create_info`
-===================   ============================  =========  =================================================================
+Summary of supported data formats
```

Comment:
Is there any way to input this from `doc/data_formats.rst`? cc: @larsoner

Reply:
This should be possible with
```rst
.. include:: ../data_formats
   :start-after: <new-crossref-coming-after-the-title>
```

I'll give it a try